.github/workflows/ci.yml

```diff
@@ -1,14 +1,10 @@
 # This is a basic workflow to help you get started with Actions
 
-name: CI
+name: CI-Build
 
 # Controls when the action will run. Triggers the workflow on push or pull request
 # events but only for the master branch
-on:
-  push:
-    branches: [ master ]
-  pull_request:
-    branches: [ master ]
+on: [push]
 
```

Comment:
any reason why we should not build all branches?

Reply:
no. but I think there is a different workflow on the release/8 branch. may you can streamline this too